cg/services/order_validation_service/models/sample.py

```diff
@@ -1,10 +1,10 @@
-from pydantic import BaseModel, ConfigDict, Field
+from pydantic import BaseModel, Field
 
 from cg.models.orders.sample_base import NAME_PATTERN, ContainerEnum
 
 
 class Sample(BaseModel):
-    application: str
+    application: str = Field(min_length=1)
```

Comment:
Is this correct if an existing sample is used?

Reply:
Pretty certain but remains to be tested in a separate PR. The behaviour does not change for the field for now though